bin/proving-service/Cargo.toml

```diff
@@ -0,0 +1,71 @@
+[package]
+authors.workspace      = true
+description            = "Miden blockchain proving service"
+edition.workspace      = true
+homepage.workspace     = true
+keywords               = ["miden", "proving", "service"]
+license.workspace      = true
+name                   = "miden-proving-service"
+readme                 = "README.md"
+repository.workspace   = true
+rust-version.workspace = true
+version                = "0.10.0"
+
+[[bin]]
+name = "miden-proving-service"
+path = "src/main.rs"
+
+[features]
+concurrent = ["miden-tx/concurrent"]
+default    = ["concurrent"]
+
+[lints]
+workspace = true
+
+[dependencies]
+async-trait                        = "0.1"
```

Comment:
nit: for this and other similar dependencies, we usually say `{ version = "0.1" }` instead of `"0.1"`.